lib/WP_Auth0_UsersRepo.php

```diff
@@ -154,18 +154,6 @@ public function update_auth0_object( $user_id, $userinfo ) {
 		self::update_meta( $user_id, 'last_update', date( 'c' ) );
 	}
 
-	/**
-	 * Delete all Auth0 meta fields for a WordPress user.
-	 *
-	 * @param int $user_id - WordPress user ID.
-	 */
-	public function delete_auth0_object( $user_id ) {
```

Comment:
Moved this to `wp_auth0_delete_auth0_object()` to avoid an unnecessary dependency injection.